bin/vagrant

```diff
@@ -159,6 +159,31 @@ begin
     env.ui.warn(I18n.t("vagrant.general.not_in_installer") + "\n", prefix: false)
   end
 
+  # Acceptable experimental flag values include:
+  #
+  # Unset  - Disables experimental features
+  # 0      - Disables experimental features
+  # 1      - Enables all features
+  # String - Enables one or more features, separated by commas
+  if ENV["VAGRANT_EXPERIMENTAL"]
```

Comment:
Should just use the `Util::Experimental.enabled?`. If explicitly disabled with `"0"` value I don't think we need it logged. For the rest of the output I think this can be cleaned up a bit by using helpers in the util instead of direct inspection of the environment variable here.